Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h

```diff
@@ -489,9 +489,6 @@ class NetworkConnectionToWebProcess final
 #if ENABLE(WEB_RTC)
     NetworkMDNSRegister m_mdnsRegister;
 #endif
-#if HAVE(COOKIE_CHANGE_LISTENER_API)
-    HashSet<String> m_hostsWithCookieListeners;
```

Comment:
This PR is making too many changes. It is as likely to introduce new bugs as fixing the particular bug we care about. It's been days, please try and make a more targeting fix we can land today.